lib/db/schema/patch-013-014.sql

```diff
@@ -0,0 +1,54 @@
+DROP PROCEDURE `account_2`;
+DROP PROCEDURE `emailRecord_2`;
+
```

Comment:
Since we usually patch the database in production prior to the release going out, we don't drop the previous versions of each stored procedure until the following release. Can you take out these DROPs (and the corresponding re-instatement in the reverse patch) - thanks.

If we did drop them here, then the current release which is relying on them would throw a wobbly. :)